src/server/pkg/deploy/cmds/cmds.go

```diff
@@ -284,3 +292,18 @@ you used the "--static-etcd-volume" flag), the underlying volume will not be
 removed.`)
 	return []*cobra.Command{deploy, undeploy}
 }
+
+func getDefaultDashImage() string {
+	// Try to get the latest according to our pachctl version, or fall back to the encoded default
+	url := fmt.Sprintf("https://pachyderm.io/versions/dash/%v.%v/latest", version.MajorVersion, version.MinorVersion)
```

Comment:
This seems like it might get us in an awkward position if we want to ship a breaking change of the web-ui because as soon as we do then anyone with an out of date `pachctl` will be spinning up a broken version of the web-ui. How would you feel about just using the hardcoded value for now and users can override it with the flag if they want to get a more advanced version. Seems like the simpler approach for now.